guides/release/components/conditional-content.md

```diff
@@ -218,6 +229,14 @@ It looks similar to a ternary operator.
 {{if condition value1 value2}}
 ```
 
+Similarly to block `if`, you need to pay attention where inline `if` is placed.
+Inline `if` can be used only inside attribute values
```

Comment:
> Inline `if` can be used only inside attribute values of HTML elements.

That's not true.
Literally the example above shows valid usage that's unrelated to HTML attributes.
These are valid examples as well:

Conditional argument value:

```hbs
<Foo @foo={{if this.bar "bar" "baz"}} />
```

Applying a modifier conditionally:

```hbs
<button type="button" {{(if @onClick (modifier this.on "click" @onClick))}} />
```